community/kernel/src/main/java/org/neo4j/kernel/impl/api/StateHandlingStatementOperations.java

```diff
@@ -120,6 +119,22 @@ public StateHandlingStatementOperations(
     }
 
     // <Cursors>
+
+    @Override
+    public Cursor<NodeItem> loadNodeById( KernelStatement statement, long nodeId ) throws EntityNotFoundException
```

Comment:
Why those 2 methods are public at all? Did not find any usages except propagations.
Also `OperationsFacade` atm already have private methods `getNodeCursor` and `getRelationshipCursor` if we will keep those 2 methods public in `EntityReadOperations` then we should be able to delete those.